debug.mjs

```diff
@@ -0,0 +1,21 @@
+export default function Debug() {
+  return `
+/*** Debug ***/
+.debug * { outline: 2px dotted var(--debug-color, darkviolet) }
+.debug * * { outline: 2px dotted var(--debug-color, darkviolet) }
+.debug * * * { outline: 2px dotted var(--debug-color, darkviolet) }
+.debug * * * * { outline: 2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * { outline: 1px solid var(--debug-color, darkviolet) }
+.debug * * * * * * { outline: 1px solid var(--debug-color, darkviolet) }
+.debug * * * * * * * { outline: 1px solid var(--debug-color, darkviolet) }
+.debug * * * * * * * * { outline: 1px solid var(--debug-color, darkviolet) }
+.debug * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+  `
+}
```

Comment:
Ahhh shoot, is this because of Firefox missing support for `has`?

Reply:
Yes. I think I'll try it with is() to slim down the selector though. Then when Firefox 116 drops I'll move it to use has as well

Reply:
Sounds good!

Reply:
`is()` didn't help.

Reply:
oh noooo 😂

Reply:
However, I don't think the extra `*` selectors are needed? `.debug *` should do it, peep this: https://codesandbox.io/s/cranky-darkness-spxmdz?file=/styles.css

Reply:
Correct. I had different styling for the different levels but not needed now.

Reply:
Ohhhh indeed, I totally missed that